includes/class-wc-payments.php

```diff
@@ -347,6 +347,32 @@ public static function init() {
 
 		add_action( 'rest_api_init', [ __CLASS__, 'init_rest_api' ] );
 		add_action( 'woocommerce_woocommerce_payments_updated', [ __CLASS__, 'set_plugin_activation_timestamp' ] );
+
+		// Init Emails. TODO check if this is the right place to do it.
+		add_filter( 'woocommerce_email_classes', [ __CLASS__, 'add_emails' ], 11 );
+		add_filter( 'woocommerce_email_actions', [ __CLASS__, 'add_email_actions' ], 11 );
+	}
+
+	/**
+	 * Function add_emails
+	 *
+	 * @param  mixed $email_classes the email classes.
+	 * @return array
+	 */
+	public static function add_emails( $email_classes ) {
```

Comment:
The same would be true for `add_emails` and `add_email_actions` methods, as this receipt email is specific to in-person payments

Reply:
If I understand correctly, you propose to change `add_emails` to something like `add_ipp_receipt_emails `? I can change it, but I am not sure if that would make sense if we add other type of emails not related to receipts in the future. The way I see it `add_emails` is generic, you could add any kind of emails in this callback. Let me know if I am following you correctly.

Reply:
> If I understand correctly, you propose to change add_emails to something like add_ipp_receipt_emails ?

Yes.

> The way I see it add_emails is generic, you could add any kind of emails in this callback.

It could be that I didn't fully understand the purpose of this method. It seemed to me that it's required to hook the loading of the main IPP receipt email class in the `woocommerce_email_classes` filter. If this method is designed to be generic to accommodate similar future classes, maybe we can rename it accordingly or provide more description somewhere about it? (in short I am not against keeping a generic name, maybe I just found it a bit confusing :) )

Reply:
I moved the email initialization and renamed the methods in [this commit](https://github.com/Automattic/woocommerce-payments/pull/4014/commits/4412d64ebe10f86909e96fbc9b8230890fee99ca), hopefully is more clear now